<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_employes</class>
 <widget class="QDialog" name="add_employes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Employee Addition</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255,255,255);</string>
  </property>
  <widget class="QLineEdit" name="le_nom">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 1px solid;
border-color: #9B0000;
border-radius: 10px;
}</string>
   </property>
   <property name="placeholderText">
    <string>Nom..</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_mail">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 1px solid;
border-color: #9B0000;
border-radius: 10px;
}</string>
   </property>
   <property name="placeholderText">
    <string>Email..</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_id">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 1px solid;
border-color: #9B0000;
border-radius: 10px;
}</string>
   </property>
   <property name="placeholderText">
    <string>ID..</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_add">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 1px solid;
border-color: #9B0000;
border-radius: 10px;
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password..</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="time_start_add">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTimeEdit{
border: 1px solid;
border-color: #9B0000;
border-radius: 9px;
}</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="time_finish_add">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTimeEdit{
border: 1px solid;
border-color: #9B0000;
border-radius: 9px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>271</width>
     <height>271</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressources.qrc">:/new/prefix1/gfx/3516835.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="b_ajouter_employes">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(158,59,59);
font: 75 9pt &quot;Arial&quot;;
border: 2px solid rgb(158,59,59);
border-radius: 15px;
}

QPushButton:hover {
  color: #fff; 
  background-color: #a63b3b; 
  border-color: #a63b3b; 
}

QPushButton:pressed {
  color: #fff; 
  background-color: #802c2c; 
  border-color: #802c2c; 
}</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="age_add_employe">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 1px solid;
border-color: #9B0000;
border-radius: 10px;
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>Age..</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_role">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 1px solid;
border-color: #9B0000;
border-radius: 10px;
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Role..</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>le_nom</zorder>
  <zorder>le_mail</zorder>
  <zorder>le_id</zorder>
  <zorder>password_add</zorder>
  <zorder>time_start_add</zorder>
  <zorder>time_finish_add</zorder>
  <zorder>b_ajouter_employes</zorder>
  <zorder>age_add_employe</zorder>
  <zorder>le_role</zorder>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
